Source/Core/DolphinQt/Config/ControllersWindow.cpp

```diff
@@ -178,12 +177,17 @@ void ControllersWindow::CreateWiimoteLayout()
 
   for (size_t i = 0; i < m_wiimote_groups.size(); i++)
   {
-    auto* wm_label = m_wiimote_labels[i] = new QLabel(tr("Wii Remote %1").arg(i + 1));
+    auto text = (i == WIIMOTE_BALANCE_BOARD ? tr("Balance Board") : tr("Wii Remote %1").arg(i + 1));
+    auto* wm_label = m_wiimote_labels[i] = new QLabel(text);
     auto* wm_box = m_wiimote_boxes[i] = new QComboBox();
     auto* wm_button = m_wiimote_buttons[i] = new QPushButton(tr("Configure"));
 
-    for (const auto& item : {tr("None"), tr("Emulated Wii Remote"), tr("Real Wii Remote")})
-      wm_box->addItem(item);
+    if (i == WIIMOTE_BALANCE_BOARD)
```

Comment:
These conditions should be braced, since their bodies travel more than one line.

Reply:
I've removed the loops entirely, just replacing it with multiple calls to `wm_box->addItem`; at this point the loops are just clutter.  (I've also added the braces).